### ISSUE TITLE
UnicodeDecodeError - 'ascii' codec can't decode byte 0xec 

### ISSUE THREAD

**Issue by wayhome25** [jazzband/django-silk#238]
When I try to check a views.py in SQL Query menu and 
silk profiler (that contains response body with Korean language ) UnicodeDecodeError raised. 
But it still works well with  English response body. 

I've checked [middleware placement](http://silk.readthedocs.io/en/latest/troubleshooting.html#middleware) and [Issues](https://github.com/jazzband/silk/issues/85) but could't find a solution yet.

Could I get some advice on this?
Please see the details below.

```
Traceback:

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/django/core/handlers/exception.py" in inner
  42.             response = get_response(request)

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/usr/local/var/pyenv/versions/3.6.0/lib/python3.6/contextlib.py" in inner
  53.                 return func(*args, **kwds)

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/django/views/generic/base.py" in view
  68.             return self.dispatch(request, *args, **kwargs)

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/django/views/generic/base.py" in dispatch
  88.         return handler(request, *args, **kwargs)

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/silk/views/profile_detail.py" in get
  28.                 actual_line, code = _code(file_path, line_num, profile.end_line_num)

File "/usr/local/var/pyenv/versions/3.6.0/envs/nomadgram/lib/python3.6/site-packages/silk/views/sql_detail.py" in _code
  20.         for i, line in enumerate(f):

File "/usr/local/var/pyenv/versions/3.6.0/lib/python3.6/encodings/ascii.py" in decode
  26.         return codecs.ascii_decode(input, self.errors)[0]

Exception Type: UnicodeDecodeError at /silk/request/e21595e3-5e74-4fe7-a824-3480a1cc9ad7/profile/9/
Exception Value: 'ascii' codec can't decode byte 0xec in position 3048: ordinal not in range(128)
```

**Comment by avelis** [comment]
@wayhome25 This might be a character encoding/decoding issue. Is it possible to know what RDBMS system you have and what character encoding it has? My hunch is that the ascii_decoding isn't accounting for either `utf-8` characters or `utf8-mb4` characters.

**Comment by wayhome25** [comment]
@avelis Thank you for the help. I'm using PostgreSQL and I've checked my database info. 
(Is this right information what you want?)

Encoding : UTF8
Ctype : en_US.UTF-8

**Comment by avelis** [comment]
Yes I believe so. I don’t have the bandwidth at the moment to investigate further. My assumption is that the decoder might be defaulting to the wrong decoding

**Comment by Jorl17** [comment]
Hi.

We have also been facing this issue for almost a year now. We cannot pinpoint the location, but it is quite annoying. Additional info:

- Python2 code base
- It only happens when viewing "Details", but it does not happen when viewing "SQL" (we explicitly accessed the SQL URL and it worked)

Cheers

**Comment by SebCorbin** [comment]
Surely fixed by #490
